vs/libpng/scripts/makefile.linux

```diff
@@ -19,13 +19,14 @@ LIBSOREL=$(LIBSOMAJ).$(RELEASE)
 OLDSO=libpng.so
 
 # Utilities:
-AR_RC=ar rc
-CC=gcc
-MKDIR_P=mkdir -p
-LN_SF=ln -sf
-RANLIB=ranlib
-CP=cp
-RM_F=/bin/rm -f
+AR ?= ar
```

Comment:
All this is copied from `libpng` and should be fixed [there](https://github.com/glennrp/libpng).